masu/config.py

```diff
@@ -61,8 +62,14 @@ class Config(object):
 
     # Celery settings
     CELERY_BROKER_URL = f'amqp://{RABBITMQ_HOST}:{RABBITMQ_PORT}'
-    CELERY_RESULT_BACKEND = f'db+postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}'
+    # CELERY_RESULT_BACKEND = f'amqp://{RABBITMQ_HOST}:{RABBITMQ_PORT}'
+    # pylint: disable=line-too-long
+    # CELERY_RESULT_BACKEND = f'db+postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}'
 
     REPORT_PROCESSING_BATCH_SIZE = 10000
 
     AWS_DATETIME_STR_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+    # The interval for scanning for new reports.
+    # This needs to be a timedelta object.
+    REPORT_CHECK_INTERVAL = datetime.timedelta(minutes=5)
```

Comment:
Seems pretty often, thought we were going for hourly based on our last discussion. Not a blocker just was my expectation.

I also think we should drive this with an environment variable that defaults to a value if not present.

Reply:
This is the "development value" so that I'm not waiting around forever to see if the scheduled task runs. 

My plan was to reset it to something sane once I actually got the scheduled tasks running. They don't currently work. 

Once the basic async stuff is merged, I'll focus on getting scheduled tasks also working.